src/actions.js

```diff
@@ -170,14 +170,15 @@ var _ = Mavo.Actions = {
 				return;
 			}
 
+			let toNode = _.getNode(to);
+
 			if ($.type(to) == "number" && !(toNode?.collection)) {
 				// If to is a number and not a collection item, it's an index
 				[index, to] = [to];
```

Comment:
Note that this can change `to`, so now that `toNode` is set before this, they may end up getting out of sync. However, this essentially deletes `to`, so I guess all you need to do to `toNode` is setting it to `undefined` (but I haven't tried it, obvs)

Reply:
I need to look at it more carefully then. Thanks.

Reply:
Thanks for the suggestion, Lea. You were right. I committed the corresponding change.

Additionally to tests, I also checked our demos, docs, and some of my apps. I haven't found any regressions.